modules/terrain/src/terrain-loader.js

```diff
@@ -0,0 +1,29 @@
+/* global __VERSION__ */ // __VERSION__ is injected by babel-plugin-version-inline
+import loadTerrain from './lib/parse-terrain';
+
+const VERSION = typeof __VERSION__ !== 'undefined' ? __VERSION__ : 'latest';
+
+export const TerrainWorkerLoader = {
+  id: 'terrain',
+  name: 'Terrain',
+  version: VERSION,
+  extensions: ['png', 'pngraw'],
+  mimeType: 'image/png',
+  options: {
+    terrain: {
```

Comment:
@Pessimistress we should probably add a default bounds or validate they are provided. Without it I was getting `NaN`s

Reply:
Oh! Nevermind, I see you've added a sane default to be relative to the image size. Nice.

Reply:
I see `mimeType: 'image/png'`, do you know if everything still works the same with WebP compressed tiles that might have `image/webp` type? I don't know exactly where the decompression happens.

Reply:
Technically we don't need to restrain the mimeType/extensions here...